src/components/pages/coach/Anamnesis/index.jsx

```diff
@@ -831,13 +869,7 @@ class Anamnesis extends React.Component{
         if(globalState.logged && !globalState.coach) return <Redirect to="/dashboard"/> 
 
         // Check if the state has been set in the parent component
-        if(location.state === undefined){
-            return <Redirect to="/coach"/> 
-        } else {
-            if(location.state.user === undefined){
-                return <Redirect to="/coach"/> 
-            }
-        }
+
 
         console.log(this.state);
```

Comment:
# please remove `console` (no-console)

In JavaScript that is designed to be executed in the browser, it's considered a best practice to avoid using methods on `console`. Such messages are considered to be for debugging purposes and therefore not suitable to ship to the client. In general, calls using `console` should be stripped before being pushed to production.

```js
console.log("Made it here.");
console.error("That shouldn't have happened.");
```

Examples of **incorrect** code for this rule:

```js
/* eslint no-console: "error" */

console.log("Log a debug level message.");
console.warn("Log a warn level message.");
console.error("Log an error level message.");
```

Reply:
Noted.
However, the console functions do not exist without reason. It is a really great, fully integrated development tool.